spec/registry/unicode.md

```diff
@@ -0,0 +1,63 @@
+# MessageFormat 2.0 Unicode Registry
+
+The `u:` namespace is reserved for use by the Unicode Consortium.
+
+## Options
+
+This section describes common options which each implementation SHOULD support
+for all _functions_ and _markup_.
+
+### `u:id`
+
+A string value that is included as an `id` or other suitable value
+in the formatted parts for the _placeholder_,
+or any other structured formatted results.
+
+Ignored when formatting a message to a string.
+
+Accepts string values, or values which can be stringified without error.
```

Comment:
Should this say it must have a value? Is the empty literal, for example, acceptable?

Reply:
An empty literal is an empty string, and is allowed by this language. This is intentional, as it's possible for a message containing a `u:id=$x` to sometimes need formatting without an identifier value, for which an empty string is a good stand-in.